lib/galaxy/managers/collections.py

```diff
@@ -487,9 +487,9 @@ def __init_rule_data(self, elements, collection_type_description, parent_identif
                 data.append([])
                 sources.append({"identifiers": identifiers, "dataset": element_object})
             else:
-                child_collection_type = collection_type_description.child_collection_type()
+                child_collection_type_description = collection_type_description.child_collection_type_description()
                 element_data, element_sources = self.__init_rule_data(
-                    element_object.elements, child_collection_type, identifiers
+                    element_object.elements, child_collection_type_description, identifiers
```

Comment:
It's a minor nitpick and maybe a bit annoying, but it would probably help to use the function keywords more often. I guess there's a chance we'd have noticed that something doesn't match up seeing `collection_type_description=child_collection_type ` .